openwisp-monitoring/luaformat.config

```diff
@@ -1,4 +1,4 @@
-column_limit: 86
+column_limit: 88
```

Comment:
I have to change `column_limit` of the formatter and that introduces changes in other files